services/cinema_hall.py

```diff
@@ -0,0 +1,18 @@
+from django.db.models import QuerySet
+
+import init_django_orm  # noqa: F401

```

Comment:
One more question, why do you need this?

Reply:
I thought, I have to import it by default, just like in video:
https://www.youtube.com/watch?v=jGNScD2_RRc&t=52s

Reply:
You need this if you wont to work with models now it is just functions
Remove this import